client/src/components/AddVideoForm.js

```diff
@@ -0,0 +1,32 @@
+import { useRef } from "react";
+
+const AddVideoForm = ({ handleAddVideo }) => {
+  const titleRef = useRef(null);
+  const urlRef = useRef(null);
+
+  return (
+    <form>
+      <label htmlFor="movie-title">
+        Title
+        <input ref={titleRef} type="text" name="movie-title" id="movie-title" />
+      </label>
+      <label htmlFor="movie-url">
+        URL
+        <input ref={urlRef} type="url" name="movie-url" id="movie-url" />
+      </label>
+      <br />
+      <button>CANCEL</button>
```

Comment:
How come this button doesn't have any click handler?

Reply:
I'm still not sure if I want to have that button, that's why it doesn't have a click handler :)

Reply:
I'll remove it for now then 👍